### PR TITLE
Create karma file if not present at bot boot, fixes #30

### LINKED ISSUE #30: Create karma file if not there on startup

### PR DESCRIPTION
**Pull request by wrideout**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -210,6 +209,8 @@ def _load_karma_from_json_file(self):
         karma_file = Path(self.karma_file_path)
         if not karma_file.is_file():
             print('No existing file found. Will start fresh.')
+            with open(self.karma_file_path, 'w') as file_ptr:
+                file_ptr.write('[]')
```
Comment (subecho) [inline bot.py:213]: Do we want to write `[]` into the file or do we just want to effectively "touch" the file to make it exist and let the format of the file be handled by the Karma events?

**Review by subecho — CHANGES_REQUESTED** [review]
Good start, but we should see about actually putting information into the file instead of just creating it and leaving it empty.

File: bot.py (hunk 1 above)
Comment (wrideout) [reply bot.py:213]: We need the empty array, otherwise it's not parsable as JSON.

Comment (wrideout) [reply bot.py:213]: This is at boot, right?  What information do you suggest we insert here?  AFAIK there won't be any new karma items (yet) to record.

**Pull request merged**